src/witan/models/dem/ccm/models.clj

```diff
@@ -102,37 +102,37 @@
      :witan/type :input
      :witan/fn :ccm-core-input/in-hist-dom-in-migrants
      :witan/params
-     {:src (with-gss "witan.models.demography/migration/domestic_in_migrants")}}
+     {:src (with-gss "witan.models.demography/migration/historic_migration_flows_domestic_in")}}
     {:witan/name :in-hist-dom-out-migrants
      :witan/version "1.0.0"
      :witan/type :input
      :witan/fn :ccm-core-input/in-hist-dom-out-migrants
      :witan/params
-     {:src (with-gss "witan.models.demography/migration/domestic_out_migrants")}}
+     {:src (with-gss "witan.models.demography/migration/historic_migration_flows_domestic_out")}}
     {:witan/name :in-hist-intl-in-migrants
      :witan/version "1.0.0"
      :witan/type :input
      :witan/fn :ccm-core-input/in-hist-intl-in-migrants
      :witan/params
-     {:src (with-gss "witan.models.demography/migration/international_in_migrants")}}
+     {:src (with-gss "witan.models.demography/migration/historic_migration_flows_international_in")}}
     {:witan/name :in-hist-intl-out-migrants
      :witan/version "1.0.0"
      :witan/type :input
      :witan/fn :ccm-core-input/in-hist-intl-out-migrants
      :witan/params
-     {:src (with-gss "witan.models.demography/migration/international_out_migrants")}}
+     {:src (with-gss "witan.models.demography/migration/historic_migration_flows_international_out")}}
     {:witan/name :in-hist-popn
      :witan/version "1.0.0"
      :witan/type :input
      :witan/fn :ccm-core-input/in-hist-popn
      :witan/params
-     {:src (with-gss "witan.models.demography/core/historic-population")}}
+     {:src (with-gss "witan.models.demography/core/historic_population")}}
     {:witan/name :in-future-mort-trend
      :witan/version "1.0.0"
      :witan/type :input
      :witan/fn :ccm-core-input/in-future-mort-trend
      :witan/params
-     {:src (with-gss "witan.models.demography/mortality/death_improvement")}}
+     {:src "witan.models.demography/mortality/future_mortality_trend_assumption.csv.gz"}}
```

Comment:
Actually, they all have .csv.gz extension, but it's applied using the `with-gss` [function](https://github.com/MastodonC/witan.models.demography/blob/787c69bb66e95d376e1092ca73178ba2eec1ac87/src/witan/models/dem/ccm/models.clj#L13)